include/velocypack/Collection.h

```diff
@@ -80,6 +82,8 @@ class Collection {
 
   static bool contains(Slice const& slice, Slice const& other);
 
+  static bool containsObject(Slice const& slice, Slice const& other);
```

Comment:
should this be renamed to `containsSubobject`?

Reply:
I checked and I actually don't have any use for this method, so I removed it for now.

I was also thinking that it is rather confusing to have array and object methods in the same class as they are not compatible, e.g. it is not possible to filter() or concat() an object, so it would make sense to either add support for objects to all methods or create an entire separate class for handling the same operations for objects.